scaleway/resource_rdb_acl.go

```diff
@@ -218,6 +222,31 @@ func rdbACLExpand(data []interface{}) ([]*rdb.ACLRuleRequest, error) {
 	return res, nil
 }
 
+func rdbACLRulesFlattenFromSchema(rules []*rdb.ACLRule, dataFromSchema []interface{}) []map[string]interface{} {
+	var res []map[string]interface{}
+	ruleMap := make(map[string]*rdb.ACLRule)
+	for _, rule := range rules {
+		ruleMap[rule.IP.String()] = rule
+	}
+
+	for _, ruleFromSchema := range dataFromSchema {
+		currentRule := ruleFromSchema.(map[string]interface{})
+		ip, err := expandIPNet(currentRule["ip"].(string))
+		if err != nil {
+			return nil // append errors
+		}
+
+		aclRule := ruleMap[ip.String()]
```

Comment:
What happens in the case `aclRule` is nil here? (Zero value of the type of the values of `ruleMap`

Reply:
Does it mean that It can be added by the other channel like the CLI and then try to read the ACL rules? I should definitely test that